caproto/curio/client.py

```diff
@@ -237,7 +238,7 @@ def __init__(self):
 
         # UDP socket broadcasting to CA servers
         self.udp_sock = ca.bcast_socket(socket)
-        self.broadcaster.our_address = self.udp_sock.getsockname()[:2]
+        self.broadcaster.our_address = safe_getsockname(self.udp_sock)[:2]
```

Comment:
@ke-zhang-rd Do you recall why we have `[:2]` everywhere this is used? Doesn't this _always_ return a `(host, port)` tuple?

Reply:
I remembered I get this from some online examples. However, it's necessary. Check [here].(https://github.com/python/cpython/blob/master/Lib/socket.py#L633-L634)

Reply:
Thanks for the reference, very helpful. CA is not compatible with IPv6; if we encounter IPv6, other, more essential aspects of caproto will break. I should have asked for clarification on this in the initial review. These should all be removed.

Reply:
I'm ok.

Reply:
Alright. All removed in 25cbb62. Confirmed:

```
$ git grep -e "\[:2\]"
caproto/_commands.py:        return DBR_INT.from_buffer(bytearray(self.buffers[0])[:2]).value
```